### ISSUE TITLE
request: command to print out the entity string name for the entity you are aiming at

### ISSUE THREAD

**Issue by trollworkout** [stanhebben/MineTweaker3#261]
kinda like the command for block info but this one is for entity only.
 i don't mean the localized string not the full entity path in the code but the actual string that the entity is registered as 
for example

full string path in the code: com.author.witchery.entities.BabaYaga
localized name: witchery.Baba Yaga.name = Baba Yaga
entity name: witchery.Baga Yaga < this one

**Comment by Yulife** [comment]
The name would be similar to the /mt blockinfo one, namely /mt entityinfo as blockinfo is rightclicking a block to get its information :)

**Comment by trollworkout** [comment]
yes:) but i don't think there is a /mt entityinfo option. is there?

**Comment by Yulife** [comment]
Not yet.
